packages/block/src/block.ts

```diff
@@ -51,6 +51,7 @@ export class Block {
   public readonly transactions: TypedTransaction[] = []
   public readonly uncleHeaders: BlockHeader[] = []
   public readonly withdrawals?: Withdrawal[]
+  public readonly requests?: CLRequestType<any>[]
```

Comment:
Isn't `ELRequestType` be more appropriate here? :thinking:

Reply:
My weakly held opinion is that this should be `CLRequests` since we're making a request to the Consensus Layer, more so than "from" the EL.

Reply:
Yeah, would agree after reading the EIP

Reply:
yes CL requests sound more appropriate, or we can call it, ELTriggeredCLRequests if we want to be more verbose